src/commands/synthetics/utils.ts

```diff
@@ -36,7 +36,7 @@ const PUBLIC_ID_REGEX = /^[\d\w]{3}-[\d\w]{3}-[\d\w]{3}$/
 const SUBDOMAIN_REGEX = /(.*?)\.(?=[^\/]*\..{2,5})/
 
 const template = (st: string, context: any): string =>
-  st.replace(/{{([A-Z_]+)}}/g, (match: string, p1: string) => (p1 in context ? context[p1] : match))
+  st.replace(/{{\s*([A-Z_]+)\s*}}/g, (match: string, p1: string) => (p1 in context ? context[p1] : match))
```

Comment:
This RegEx seems already available in synthetics common: https://github.com/DataDog/synthetics-worker/blob/prod/common-public/src/variable.ts#L1
Though, it's way more permissive, so I am not sure we should use it here, but we might want to synchronize the two, no?

Reply:
We don't use synthetics-common in datadog-ci but we should certainly synchronize them yes